vulcan/lib/client/jsx/components/workflow/session/primary_inputs.tsx

```diff
@@ -59,19 +78,39 @@ function PrimaryInputsInner() {
     }, {} as {[k: string]: BoundInputSpecification[]});
   }, [inputSpecifications, inputs, setInputs, state.data, state.session, state.status, workflow]);
 
+  const [ expanded, setExpanded ] = useState('');
+
+  const [ open, setOpen ] = useState(false);
+
+  const classes = useStyles();
+
   return (
-    <div className='primary-inputs'>
-      {Object.keys(groupedInputs)
-        .sort()
-        .map((groupName, index) => {
-          return (
-            <InputGroup
-              groupName={groupName}
-              key={index}
-              inputs={groupedInputs[groupName]}
-            />
-          );
-        })}
-    </div>
+    <Card className={classes.card}>
+        <Grid className={classes.header} container alignItems='center' justify='space-between'>
+          <Grid item>
+            <Typography variant='h6'>Inputs</Typography>
+          </Grid>
+          <Grid item>
+            <IconButton size='small' onClick={ () => setOpen(!open) }>
```

Comment:
super, super-nit: On the other drawers, it seems like clicking on the heading itself seems to open / close the drawer (not just the icon). Should we be consistent and put this onClick handler here on the grandparent `Grid`?